app/models/requests/requestable_decorator.rb

```diff
@@ -123,14 +123,18 @@ def status_badge
                   else
                     "badge-danger"
                   end
-      status = if requestable.status_label.nil? || requestable.status == requestable.status_label
-                 requestable.status
-               else
-                 requestable.status + ' - ' + requestable.status_label
-               end
+      status = status_for_badge
```

Comment:
Is this assignment needed? (couldn't we just call status_for_badge on the next line?)

Reply:
So true. I feel silly.